templates/client-daemonset.yaml

```
@@ -121,12 +121,12 @@ spec:
             - name: ADVERTISE_IP
               valueFrom:
                 fieldRef:
-                  {{- if not .Values.client.exposeGossipPorts }}
-                  fieldPath: status.podIP
-                  {{- else }}
-                  # Clients will be exposed on their node's hostPort for external-to-k8s communication,
-                  # so they need to advertise their host ip instead of their pod ip.
+                  {{- if .Values.client.exposeGossipPorts }}
+                  {{/* Clients will be exposed on their node's hostPort for external-to-k8s communication,
```

```suggestion
                  {{- /* Clients will be exposed on their node's hostPort for external-to-k8s communication,
```

Otherwise you get a newline